static/app/views/releases/detail/overview/releaseAdoption.tsx

```diff
@@ -198,67 +204,97 @@ function ReleaseComparisonChart({
     allowEmptyPeriod: true,
   });
 
+  const adoptionStage = release.adoptionStages?.[project.slug].stage;
+  const adoptionStageLabel = ADOPTION_STAGE_LABELS[adoptionStage];
+  const allEnvironments = environment.length === 0;
+  const multipleEnvironments = environment.length > 1;
+
   return (
-    <RelativeBox>
-      <ChartLabel top="0px">
-        <ChartTitle>
-          {t('Sessions Adopted')}
-          <QuestionTooltip
-            position="top"
-            title={t(
-              'Adoption compares the sessions of a release with the total sessions for this project.'
-            )}
-            size="sm"
+    <div>
+      <SidebarSection
+        title={
+          allEnvironments
+            ? t('All Environments')
+            : multipleEnvironments
+            ? t(`${environment.join(', ')}`)
+            : t(`${environment}`)
+        }
+      >
+        <SidebarSectionTitle title={t('Adoption Stage')}>
```

Comment:
This is a bit confusing component architecture to me.
`<SidebarSection title="">`
and
`<SidebarSectionTitle title="">`
are too similar.